### PR TITLE
Add 16 Heads As A Test Notebook

### PR DESCRIPTION
**Pull request by ArthurConmy**

### CODE REVIEW

File: acdc/acdc_utils.py [hunk 1]
```diff
@@ -181,9 +183,18 @@ def kl_divergence(
         assert kl_div.shape == mask_repeat_candidates.shape, (kl_div.shape, mask_repeat_candidates.shape)
         denom = mask_repeat_candidates.long().sum()
         kl_div = kl_div * mask_repeat_candidates
-        answer = kl_div.sum() / denom
+        if return_one_element:
```
Comment (rhaps0dy) [inline acdc/acdc_utils.py:186]: Why is this needed? Seems like a thing that makes mistakes easy?

File: acdc/acdc_utils.py [hunk 2]
```diff
@@ -181,9 +183,18 @@ def kl_divergence(
...
+        else:
+            answer = kl_div
     else:
-        answer = kl_div.mean()
+        if return_one_element:
+            answer = kl_div.mean()
+        else:
+            answer = kl_div
+
+    if not return_tensor:
```
Comment (rhaps0dy) [inline acdc/acdc_utils.py:196]: Seems easier to just call `.item()` on the output of `kl_divergence` if it's needed

**Review by rhaps0dy — APPROVED** [review]

File: acdc/TLACDCExperiment.py [hunk 3]
```diff
@@ -132,7 +132,7 @@ def __init__(
                 config=wandb_config,
             )
 
-        self.metric = metric
+        self.metric = lambda x: metric(x).item()
```
Comment (ArthurConmy) [inline acdc/TLACDCExperiment.py:135]: New way of using both tensors and floats through the code

**Comment by ArthurConmy** [comment]
@rhaps0dy agreed that passing round that flag is annoying! 

I removed it and added the new way of using both tensors and floats, how does that sound?

**Comment by rhaps0dy** [comment]
Sounds great, hit that merge button!
